docs/about/features_index/fed_eval.rst

```diff
@@ -24,34 +24,489 @@ In general pipeline is as follows:
 Example Using the Task Runner API (Aggregator-based Workflow)
 --------------------------------------------------------------
 
-To demonstrate usage of the task runner API (aggregator-based workflow) for federated evaluation, consider the `Hello Federation example <https://github.com/securefederatedai/openfl/blob/develop/tests/github/test_hello_federation.py>`_. This sample script creates a simple federation with two collaborator nodes and one aggregator node, and executes based on a user specified workspace template. We provide a ``torch_cnn_mnist_fed_eval`` template, which is a federated evaluation template adapted from ``torch_cnn_mnist``. 
+The following steps can be leveraged to achieve practical e2e usage of FedEval
 
-This script can be directly executed as follows:
+*N.B*: We will be using torch_cnn_mnist plan itself for both training and with some minor changes for evaluation as well
+
+*Prerequisites*: Please ensure that latest OpenFL version>1.6 is installed, you can also choose to install from source.
+
+With latest OpenFL version>1.6 aggregator start command is enhanced to have an optional argument '--task_group' which, 

```

Comment:
To avoid someone mis-interpreting this and trying with 1.6, I suggest either referring to 1.7 and beyond, or just saying "the latest" version.